apps/react-workshop/src/ProgressLinear.stories.tsx

```diff
@@ -55,10 +56,25 @@ export const Negative = () => {
       labels={[
         'Upload failed',
         <Icon key='icon'>
-          <SvgStatusErrorHollow />
+          <SvgStatusError />
         </Icon>,
       ]}
       status='negative'
     />
   );
 };
+
+export const Warning = () => {
+  return (
+    <ProgressLinear
+      value={45}
```

Comment:
Value should be 100 for this example to make sense?

Reply:
done!